### ISSUE TITLE
Add ability to sign up once a user is past Hero

### ISSUE THREAD

**Issue by yaboi** [BeardandFedora/writeon.io#5]
Number 4 on "An Epic List of 100 Growth Hacks"

> 1. Sign-up Functionality that Stays at the Top
>    Use a hello bar style fixed position sign-up form.

See: http://www.searchenginejournal.com/epic-list-100-growth-hacks-startups/118690/

---

Either we can make the current navigation opaque, then show a "sign up" button when a user scrolls down or keep navigation static and introduce a secondary navigation that shows when a user scrolls.

**Comment by thinq4yourself** [comment]
I like the idea that when the user scrolls, show an opaque nav/header that has some minimalist CTA to signup <section> at bottom of page.

**Comment by yaboi** [comment]
I like the idea of the header showing up after a certain scrollHeight. Do you think the "header/nav" should be fixed to the bottom of the page, though?

Examples?

**Comment by thinq4yourself** [comment]
No, I am not in cahoots with a fixed to bottom nav, unless it was related to boosting signups. Unless, "nav" is misleading here. Now I think examples would be best. Have any?

Best example of getting people to signup I can think of right now is:

http://www.crazyegg.com (wait for it....)

**Comment by thinq4yourself** [comment]
Then bring them to /home, as it is. 

Thoughts?

**Comment by yaboi** [comment]
"nav" wasn't misleading, the "...CTA to signup at bottom of page." was misleading. I get what you mean now. : )

The dreaded popup. The only thing I hate more than it's intrusiveness is that it ACTUALLY works! Ugh. Whelp, let's give it a shot. Let's A/B the functionality...?

~Beard

**Comment by yaboi** [comment]
I like the idea of having a sticky top-nav that has one CTA for signing up (and logo on left, of course).

I'll work on implementing this.
